lib/spack/docs/packaging_guide.rst

```diff
@@ -4048,6 +4048,69 @@ File functions
 :py:func:`touch(path) <spack.touch>`
   Create an empty file at ``path``.
 
+.. _make-package-findable:
+
+--------------------------------------------------------
+Making a package discoverable with "spack external find"
```

Comment:
Question (I haven't generated the docs and looked): does this look ok in a  browser when you load the docs?  I'm worried this will be hard to read in the readthedocs theme.

Also I think it should be `` instead of " around `spack external find` (for consistency w/other titles)

Reply:
> does this look ok in a browser when you load the docs? I'm worried this will be hard to read in the readthedocs theme.

Do you mean because the title is long? It renders fine for me. It also looks fine when I apply your suggestion to use ``` ``spack external find`` ``` instead of `"spack external find"`

Reply:
ok cool!  resolved.